hydra/core/override_parser/types.py

```diff
@@ -146,6 +146,8 @@ def __eq__(self, other: Any) -> Any:
 # to support recursive type definitions.
 ElementType = Union[str, int, float, bool, List[Any], Dict[str, Any]]
 ParsedElementType = Optional[Union[ElementType, QuotedString]]
+ElementTransformerType = Callable[[ParsedElementType], Optional[ElementType]]
```

Comment:
cc @omry checking if this name (for the transformer type) looks alright

Reply:
```suggestion
TransformaerType= Callable[[ParsedElementType], Optional[ElementType]]
```